blog/2018-12-04-cache-invalidation-with-change-data-capture.adoc

```diff
@@ -0,0 +1,490 @@
+= Automating Cache Invalidation With Change Data Capture
+gmorling
+:awestruct-tags: [ discussion, examples ]
+:awestruct-layout: blog-post
+
+The https://docs.jboss.org/hibernate/stable/orm/userguide/html_single/Hibernate_User_Guide.html#caching-config[second-level cache] of Hibernate ORM / JPA is a proven and efficient way to increase application performance:
+caching read-only or rarely modified entities avoids roundtrips to the database,
+resulting in improved response times of the application.
+
+Unlike the first-level cache, the second-level cache is associated to the session factory (or entity manager factory in JPA terms),
```

Comment:
`associated with` instead of `associated to`?

Reply:
Done.